src/pages/GE_TokenizeCard.page

```diff
@@ -10,9 +11,7 @@
         </form>
     </article>
 
-    <!-- TODO: Update src with production url and/or upload as static resource -->
-    <script src="https://dmjphp3j3je7i.cloudfront.net/elevate.min.js"></script>
-
+    <script src="{!elevateSDKURL}"></script>
     <script>
         const hostname = window.location.hostname;
         const arr = hostname.split(".");
```

Comment:
We should also replace the "clientId: 'bees' down on line 27 with a dynamic reference to the "productId" we get passed through elevate config

Reply:
👍🏽

Reply:
fixed!